### ISSUE TITLE
Map internals of models code-fix appears in non-empty method

### ISSUE THREAD

**Issue by TonySkorik** [DeagleGross/SharpCoachPlugin#22]
If the code fix menu is called in a non-empty method it has the `Map internals of models` code-fix (see the screenshots section). 
Looks like this menu item should not be shown in non-empty method context.

**To Reproduce**
1. Add the following method (the types in the method body don't actually matter)
```
private async Task BuildResponse(Dictionary<string, List<string>> keyValueItemsByTables)
{
    var responseString = "some string";

    List<object> retBatches = new(keyValueItemsByTables.Count);

    foreach (var keyValueItemsForTable in keyValueItemsByTables)
    {
        // some code
    }
}
```
2. Place cursor on the first method line and call the lightbulb menu.
3. Observe the `Map internals of models` code-fix.

**Expected behavior**
It seems that the code fix should not be added in the lightbulb menu in the non-empty method body.

**Screenshots**
![image_2023-03-03_14-37-48](https://user-images.githubusercontent.com/5507889/222702913-85f69544-5aee-45f9-9455-70018919258e.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 22H2 Build 19045.2604
 - Rider version and build number : #2022.3.2 Build #RD-223.8617.53, built on January 25, 2023

**Comment by TonySkorik** [comment]
Correct approach would be to restrict code-fix appearance only to the method signature and avoid showing it if code fixes menu is called from within the method itself;

**Comment by DeagleGross** [comment]
thanks for submitting an issue, agree with the lattest comment - it makes sense to block intellisence suggestion when cursor is pointing to any other place except the method name.

will do when I find time

**Comment by DeagleGross** [comment]
refactored to use this approach, rather than checking if the method body is empty
```
Correct approach would be to restrict code-fix appearance only to the method signature and avoid showing it if code fixes menu is called from within the method itself;
```

will be available in 1.3.1 version soon
